hadoop-hdds/docs/content/interface/Cli.md

```diff
@@ -139,6 +139,26 @@ $ ozone sh volume list /
   ....
 } ]
 ```
+
+If the volume is empty, we can delete volume using below command. 
+
+```shell
+$ ozone sh volume delete /vol1
+Volume vol1 is deleted
+```
+
+If the volume contains some buckets/keys, we can delete volume recursively. Which will delete all the buckets, keys for the specified volume and also volume will be deleted.
```

Comment:
```suggestion
If the volume contains any buckets or keys, we can delete the volume recursively. This will delete all keys and buckets within the volume, and then delete the volume itself.
```